app/helpers/macros/render.py

```diff
@@ -0,0 +1,4 @@
+import app.tasks.highlight
+
+def highlight_answer(answer):
+    return app.tasks.highlight.syntax_highlight(answer.code, answer.get_language().get_hljs_id())
```

Comment:
That's not how Celery tasks are called when they're sent to task workers - you use `task.delay(args).wait()`.